src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/rectangle.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, 0.6);
+$mainColor: $white;
+$boxShadow: 0 0 20px 0 rgba($black, 0.3);
+$borderSize: 2px;
+
+.rectangle {
+    position: absolute;
+    background: transparent;
+    border: $borderSize solid $mainColor;
+    z-index: 1000;
+    cursor: move;
+    box-sizing: content-box;
+    overflow: visible;
+
+    &::after {
+        content: "";
+        position: absolute;
+        top: -$borderSize;
+        left: -$borderSize;
+        bottom: -$borderSize;
+        right: -$borderSize;
+        outline: 20000px solid $backdropColor;
```

Comment:
Is that magic number necessary? Why can't we simply set `top`, `left`, `bottom` and  `right` to `0` to stretch the element?